scripts/generic-ci-tests.sh

```diff
@@ -175,20 +175,26 @@ case "$TEST_SUITE" in
     "bok-choy")
 
         PAVER_ARGS="-n $NUMBER_OF_BOKCHOY_THREADS"
+        if [[ -n "$WHO_TESTS_WHAT" ]]; then
+            PAVER_ARGS="$PAVER_ARGS --with-wtw"
+        fi
+        if [[ -n "$PYTEST_CONTEXTS" ]]; then
+            PAVER_ARGS="$PAVER_ARGS --pytest-contexts --coveragerc=common/test/acceptance/.coveragerc"
```

Comment:
Why does this change the coveragerc?

Reply:
The .coveragerc isn't set in bokchoy normally.